cmd/bosun/sched/check.go

```diff
@@ -40,7 +40,14 @@ func NewStatus(ak expr.AlertKey) *State {
 	}
 }
 
-func (s *Schedule) Status(ak expr.AlertKey) *State {
+func (s *Schedule) GetStatus(ak expr.AlertKey) *State {
```

Comment:
Sadly, no. The web package is the only user of GetStatus and GetOrCreateStatus, and both of them need the full structure, as far as I can tell. Our test rule page does some crazy things.